platform/solaris/ips/create.sh

```diff
@@ -185,33 +194,68 @@ do
    PKG pkgsend add dir mode=0755 owner=root group=bin path=${dir}
 done
 
-for dir in /var/opengrok /var/opengrok/data /var/opengrok/etc \
-           /var/opengrok/log /var/opengrok/src
+PKG pkgsend add dir mode=0755 owner=webservd group=webservd path=/var/opengrok
```

Comment:
This will be created automatically by the migration script. In fact I think `pkglint` would complain if the package delivered content under `/var/share`.